ejb3/src/main/resources/schema/jboss-ejb-timer-service_2_0.xsd

```diff
@@ -36,25 +36,25 @@
     <xs:complexType name="timer-serviceType">
         <xs:complexContent>
             <xs:extension base="javaee:jboss-assembly-descriptor-bean-entryType">
-                <xs:all>
-                    <xs:choice minOccurs="0">
-                        <xs:element name="persistence-store-name" type="xs:string">
+                <xs:choice>
+                    <xs:element name="persistence-store-name" type="xs:string" maxOccurs="1">
+                        <xs:annotation>
+                            <xs:documentation>Identifies the name of the data store to use for persistent timers.</xs:documentation>
+                        </xs:annotation>
+                    </xs:element>
+                    <xs:sequence>
+                        <xs:element name="persistent-timer-management" type="xs:string" maxOccurs="1">
                             <xs:annotation>
-                                <xs:documentation>Identifies the name of the data store to use for persistent timers.</xs:documentation>
+                                <xs:documentation>Identifies the timer-management provider to use for persistent timers.</xs:documentation>
                             </xs:annotation>
                         </xs:element>
-                        <xs:element name="persistent-timer-management" type="xs:string">
+                        <xs:element name="transient-timer-management" type="xs:string" maxOccurs="1">
                             <xs:annotation>
                                 <xs:documentation>Identifies the timer-management provider to use for persistent timers.</xs:documentation>
```

Comment:
for persistent timers - > for transient ...

Reply:
indeed. :/